File: official/docs/python/v7/orders/one-call-buy.py
import easypost
import os

easypost.api_key = os.getenv("EASYPOST_API_KEY")

order = easypost.Order.create(
    carrier_accounts=["ca_..."],
    service="NextDayAir",
    to_address={"id": "adr_..."},
    from_address={"id": "adr_..."},
    shipments=[
        {
            "parcel": {
                "predefined_package": "FedExBox",
                "weight": 10.2,
            }
        },
        {
            "parcel": {
                "predefined_package": "FedExBox",
                "weight": 17.5,
            }
        },
    ],
)

print(order)

File: official/docs/node/v7/orders/one-call-buy.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const order = await client.Order.create({
    carrier_accounts: ['ca_...'],
    service: 'NextDayAir',
    to_address: 'adr_...',
    from_address: 'adr_...',
    shipments: [
      {
        parcel: {
          weight: 10.2,
        },
      },
      {
        parcel: {
          predefined_package: 'FedExBox',
          weight: 17.5,
        },
      },
    ],
  });

  console.log(order);
})();
